app/assets/stylesheets/styles/components/guides/components/_large_screen_show_top.css.scss

```diff
@@ -0,0 +1,31 @@
+.large-guide {
+  padding: 20px;
+
+  .canopy {
+    margin-top: 20px;
+  }
+
+  .guide-controls {
+    margin-right: 0px;
+  }
+
+  .guide-name {
+    display: inline-block;
+  }
+
+  .header {
+    border-bottom: rgb(148, 148, 148) 3px solid;
+  }
+
+  .crop-name {
+    margin-right: 20px;
+  }
+
+  .guide-relationships {
+    color: #666666;
```

Comment:
Color literals like `#666666` should only be used in variable declarations; they should be referred to via variable everywhere else.